src/streaming-client.ts

```diff
@@ -55,10 +54,14 @@ export class RevAiStreamingClient extends EventEmitter {
             readableObjectMode: true,
             writableObjectMode: true
         });
-        this.client = new client({
-            // @ts-ignore
+        const clientConfig: IClientConfig = {};
+        const clientConfigExtensionProperties = {
```

Comment:
I liked the interface separation. The compiler was complaining after I updated the typescript version.